compute/cker/include/cker/operation/BinaryArithmeticOps.h

```diff
@@ -51,13 +51,7 @@ const std::function<T(const T &, const T &)> GetBinaryArtithmeticFn(BinaryArithm
     }
     case BinaryArithmeticOpType::DIV:
     {
-      return [](const T &a, const T &b) -> T {
-        if (b == 0)
-        {
-          throw std::runtime_error("Divide by zero");
-        }
-        return a / b;
-      };
+      return [](const T &a, const T &b) -> T { return a / b; };
```

Comment:
What about

```suggestion
      return divide;
```

And have divide function as a static or something.

```cpp
template<T>
T divide(const T& a, const T &b)
{
        if (b == 0)	
        {	
          throw std::runtime_error("Divide by zero");	
        }	
        return a / b;	
      };
}

template <>
float divide(const float &a, const float &b)
{
  return a/b;
}
```

Reply:
@wateret I've tried, but I got 
```
In file included from /home/brian/z/ONE/runtime/onert/core/src/interp/operations/BinaryArithmeticOps.cc:17:
/home/brian/z/ONE/compute/cker/include/cker/operation/BinaryArithmeticOps.h:44:1: error: type qualifiers ignored on function return type [-Werror=ignored-qualifiers]
   44 | const float divide(const float &a, const float &b)
      | ^~~~~
/home/brian/z/ONE/compute/cker/include/cker/operation/BinaryArithmeticOps.h:44:13: error: template-id ‘divide<>’ for ‘const float nnfw::cker::divide(const float&, const float&)’ does not match any template declaration
   44 | const float divide(const float &a, const float &b)
      |             ^~~~~~
/home/brian/z/ONE/compute/cker/include/cker/operation/BinaryArithmeticOps.h:34:3: note: candidate is: ‘template<class T> T nnfw::cker::divide(const T&, const T&)’
   34 | T divide(const T& a, const T &b)`
```

Reply:
Fixed the comment!